src/main/java/de/dennisguse/opentracks/adapters/IntervalStatisticsAdapter.java

```diff
@@ -59,8 +59,9 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
 
         viewHolder.rate.setText(StringUtils.formatSpeed(context, interval.getSpeed(), metricUnits, isReportSpeed));
 
-        viewHolder.gain.setText(StringUtils.formatDistance(context, Distance.of(interval.getGain_m()), metricUnits));
-        viewHolder.loss.setText(StringUtils.formatDistance(context, Distance.of(interval.getLoss_m()), metricUnits));
+        viewHolder.gain.setText(interval.hasGain() ? StringUtils.formatDistance(context, Distance.of(interval.getGain_m()), metricUnits) : "-");
```

Comment:
Should be `R.strings.value_unknown` aka `<string name="value_unknown" translatable="false">-</string>`

Reply:
Done... I didn't realize of that.